shared/src/Datadog.Trace.ClrProfiler.Native/Datadog.Trace.ClrProfiler.Native.vcxproj.filters

```diff
@@ -84,6 +84,18 @@
     <ClInclude Include="resource.h">
       <Filter>Header Files</Filter>
     </ClInclude>
+    <ClInclude Include="..\..\..\shared\src\native-src\version.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="process_helper.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="single_step_guard_rails.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
```

Comment:
Do we want/need the single step stuff for this PR, or was this just accidentally missing when I added them before? :oops:

Reply:
Accidentally missing